rustls/src/error.rs

```diff
@@ -636,7 +636,10 @@ mod other_error {
     ///
     /// Enums holding this type will never compare equal to each other.
     #[derive(Debug, Clone)]
-    pub struct OtherError(#[cfg(feature = "std")] pub Arc<dyn StdError + Send + Sync>);
+    pub struct OtherError(
+        #[cfg(feature = "std")] // XXX XXX TODO ADD COMMENT TO KEEP SEPARATE LINES HERE
+        pub  Arc<dyn StdError + Send + Sync>,
```

Comment:
I have a feeling that this encountered a couple possible issues with rustfmt:

- rustfmt seems to squish #[cfg(...)] together with other lines of code here; I think rustfmt should favor keeping #[cfg(...)] on its own line in this case as well
- rustfmt also seems to put an extra space between `pub` ... `Arc` if it is on its own line here.